### ISSUE TITLE
Write documentation

### ISSUE THREAD

**Issue by tobiasvl** [eievui5/evscript#1]
This seems really interesting! Are there plans for more (or any) documentation on the VM?

**Comment by evie-calico** [comment]
Yes, the language is in pretty dire need of documentation but I've been putting it off since I'm the only user I know of ^^

Ideally I'd document the basic features of the language and provide some explanations for more unique features like the environments. The VM part would be pretty simple to document as well so I don't see that being an issue.

**Comment by ISSOtm** [comment]
I'm currently working on a mdBook-based manual. Currently, I'm planning to first show how the script syntax works, then explain what it gets compiled to, and how that's controlled via `env` blocks.
